yaptide/application.py

```diff
@@ -50,6 +54,13 @@ def create_app():
 
     return app
 
+def check_submodules():
+    with open('.gitmodules') as f:
+        for line in f:
+            if 'path' in line:
+                path = line.split('=')[-1].strip()
+                if not os.path.exists(path):
+                    return False
```

Comment:
```suggestion
                if not Path(path):
                    return False
```

why not to use pathlib ?